app/javascript/entrypoints/pdc/work_edit_file_upload.es6

```diff
@@ -63,10 +63,28 @@ export default class WorkEditFileUpload {
       headers: { 'X-CSRF-Token': tokenContent },
       bundle: true, // upload all selected files at once
       formData: true, // required when bundle: true
-      getResponseData() {
-        // Reload the file list displayed
-        const fileTable = $('#files-table').dataTable();
-        fileTable.api().ajax.reload();
+      getResponseData(filenames) {
+        const loadBalancerError = (filenames || '').toLowerCase().includes('your support id');
```

Comment:
This line is the check. If the Load Balancer blocks the request we detect it because the "filenames" will include the error message "Your Support ID..."